### PR TITLE
Add: Action for golang linting

### PR DESCRIPTION
**Pull request by y0urself**
**What**:

* Adding an action for golang linting
  * it is basically only using the golangci-lint action, since my initial research classified this as the best practice currently (there are a lot of different linters and this seems to run multiple parallel and is very configurable)

**Why**:

* Easier usage
* Unifying golang linting for greenbone projects

**How**:

* it is basically only using the golangci-lint action

**Checklist**:

- [ ] Tests
- [ ] CHANGELOG.md entry
- [ ] Documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,3 +131,21 @@ with:
     gpg-fingerprint: ${{ secrets.BAR_BAZ }}
     file: ./foo/bar
 ```
+
+## Support
+
+For any question on the usage of python-gvm please use the
+[Greenbone Community Portal](https://community.greenbone.net/c/gmp). If you
```
Comment (bjoernricks) [inline README.md:138]:
```suggestion
[Greenbone Community Forum](https://forum.greenbone.net/). If you
```

File: README.md [hunk 2]
```diff
@@ -84,3 +131,21 @@ with:
...
+found a problem with the software, please
+[create an issue](https://github.com/greenbone/gvm-tools/issues)
```
Comment (bjoernricks) [inline README.md:140]:
```suggestion
[create an issue](https://github.com/greenbone/actions/issues)
```

File: lint-golang/action.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+branding:
+  icon: "package"
+  color: "green"
+
+runs:
+  using: "composite"
+  steps:
+    - run: echo "Checkout Repository"
+      shell: bash
```
Comment (bjoernricks) [inline lint-golang/action.yml:23]: I don't think we really need that.
```suggestion
```

**Review by bjoernricks — CHANGES_REQUESTED** [review]

File: lint-golang/action.yml (hunk 3 above)
Comment (y0urself) [reply lint-golang/action.yml:23]: Probably not. ;)

**Review by bjoernricks — APPROVED** [review]

**Pull request merged**